build_wasm.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-EMCC_CFLAGS="-I/emsdk/upstream/include -L/emsdk/upstream/lib" EMCC_FORCE_STDLIBS=libc emmake make $1
+EMCC_CFLAGS="-I/emsdk/upstream/include -L/emsdk/upstream/lib -sINITIAL_MEMORY=26214400" EMCC_FORCE_STDLIBS=libc emmake make $1
```

Comment:
Why do we need this new build flag?

Reply:
Increasing initial memory, reserving bigger memory for compilation prevents the build being failed. The building for ARM seem to reserve more memory than for AMD.